### DISCUSSION TITLE
Github Web Devs

### DISCUSSION THREAD

**Discussion by AwaisOem** [community/community#131714] [Programming Help]
### Body

This discussion is for helping each other with Development and Programming.

### Guidelines

- [ ] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @AwaisOem, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share more information about what you'd like to talk about or work on specifically. Giving a few more details might help someone give you a nudge in the right direction. 😄
